tests/fixtures.py

```diff
@@ -1,18 +1,20 @@
 # Copyright (C) 2022 Mandiant, Inc. All Rights Reserved.
 
-import os
+from pathlib import Path
 
 import pytest
 
-CD = os.path.dirname(__file__)
+CD = Path(__file__).resolve().parent
 
 
 @pytest.fixture
 def exefile() -> str:
     # decode-in-place is among the fastest samples in data/src
-    return os.path.join(CD, "data", "src", "decode-in-place", "bin", "test-decode-in-place.exe")
+    path = CD / "data" / "src" / "decode-in-place" / "bin" / "test-decode-in-place.exe"
+    return str(path)
```

Comment:
return pathlib.Path

Reply:
I am not sure on how to change this code because the parser tries to convert the sample path to an argparse.FileType and does some string related operations that are not allowed with pathlib.Path objects.

Reply:
ah, ok, i understand how this might interact weirdly with pytest and/or argparse. no worries.